CMakeLists.txt

```diff
@@ -740,9 +740,14 @@ if(WITH_OPTIM)
             endif()
         endif()
     elseif(BASEARCH_X86_FOUND)
+        check_xsave_intrinsics()
```

Comment:
I think this entire XSave block should go at the end of the if statement. I've tried to keep all the checks alphabetical.

Reply:
But idk, it might be ok as is. Maybe put fallback_bultin's block above this one?

Reply:
Yeah I think it should be alphabetical, in the same order as configure script.

Reply:
This has to be before line 749, but otherwise the placement isn't critical...